plonky2/src/fri/oracle.rs

```diff
@@ -117,10 +117,24 @@ impl<F: RichField + Extendable<D>, C: GenericConfig<D, F = F>, const D: usize>
         blinding: bool,
         fft_root_table: Option<&FftRootTable<F>>,
     ) -> Vec<Vec<F>> {
-        let degree = polynomials[0].len();
+        if blinding {
```

Comment:
This was a little awkward, but we are just asserting false if someone sets zero-knowledge to true and no-random to true, since the field is not so easy to put behind a feature flag.